src/types.js

```diff
@@ -59,9 +63,70 @@ export type SpacingAliases = {
   +[spacingAlias: string]: number,
 }
 
+export type DisplayAliases = {
+  +[displayAlias: string]: string,
+}
+
 export type ConfigProviderContext = {|
   +xnReflex: {|
+    +displayAliases: DisplayAliases,
     +spacingAliases: SpacingAliases,
     +base: number,
   |},
 |}
+
+export type OneResolutionGrid = {
+  align?: AlignGrid,
+  children?: React.Node,
+  className?: string,
+  dev?: Dev,
+  justify?: Justify,
+  margin?: Spacing,
+  marginBottom?: SpacingValues,
+  marginLeft?: SpacingValues,
+  marginRight?: SpacingValues,
+  marginTop?: SpacingValues,
+  padding?: Spacing,
+  paddingBottom?: SpacingValues,
+  paddingLeft?: SpacingValues,
+  paddingRight?: SpacingValues,
+  paddingTop?: SpacingValues,
+  show?: DisplayValues,
+  size?: Size,
+  style?: { [string]: string | number },
+}
```

Comment:
`OneResolutionGrid` is used internally only, after `withResolution` has resolved the values to use

Reply:
If this is not reused, can we move it to the file where it is being used?